N/NetCDF/build_tarballs.jl

```diff
@@ -87,9 +86,14 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("HDF5_jll"),
+    Dependency("HDF5_jll", v"1.12.0"),
```

Comment:
Note that we have two ways of expressing compatibilities:
```julia
Dependency("Foo_jll", v"1.2.3")
```
means "build with `Foo_jll` v1.2.3, but it's compatible with any version, while (note: I don't exactly remember the syntax by heart)
```julia
Dependency(PackageSpec(; name = "Foo_jll", version = "1.2.3"))
```
means "this is compatible with Foo_jll v1.2.3"